lisa/util/shell.py

```diff
@@ -104,14 +101,6 @@ def wait_tcp_port_ready(
     return is_ready, result
 
 
-def _is_valid_ipv6(address: str) -> bool:
-    try:
-        ipaddress.IPv4Address(address)
-        return False
-    except ipaddress.AddressValueError:
```

Comment:
@LiliDeng It looks the logic should be changed. We need to check if it's a valid IPv6 here. Only set it to true, when it's a valid IPv6 address.